src/panels/config/network/supervisor-network.ts

```diff
@@ -667,6 +684,11 @@ export class HassioNetwork extends LitElement {
       const { ip } = parseAddress(this._interface![version]!.address![index]);
       this._interface[version]!.address![index] = formatAddress(ip, value);
       this.requestUpdate("_interface");
+    } else if (id === "prefix") {
+      const index = (ev.target as any).index as number;
+      const { ip } = parseAddress(this._interface![version]!.address![index]);
+      this._interface[version]!.address![index] = `${ip}/${value}`;
+      this.requestUpdate("_interface");
```

Comment:
This is duplicate code. 

You can check if its `prefix` or `netmask` and then just make an inline if for the interface address set.

Reply:
okay